src/ios/CDVFileTransfer.m

```diff
@@ -552,6 +552,8 @@ - (void)connectionDidFinishLoading:(NSURLConnection*)connection
 
     if (self.direction == CDV_TRANSFER_UPLOAD) {
         uploadResponse = [[NSString alloc] initWithData:self.responseData encoding:NSUTF8StringEncoding];
+        if (uploadResponse == nil)
+            uploadResponse = [[NSString alloc] initWithData: self.responseData encoding:NSISOLatin1StringEncoding];
```

Comment:
On Android Latin1 spec symbols result in � in webview.

On Windows such an exception occurs on reading the response stream:
```
Exception is about to be caught by JavaScript library code at line 178, column 37 in ms-appx://io.cordova.hellocordova/www/plugins/cordova-plugin-file-transfer/src/windows/FileTransferProxy.js

0x80070459 - JavaScript runtime error: No mapping for the Unicode character exists in the target multi-byte code page.
```